docs/source/contribute/contrib_doc.rst

```diff
@@ -0,0 +1,314 @@
+.. _contrib_doc_dev:
+
+Contributing to the Documentation
```

Comment:
Let's delete this file, we have this in scancode-toolkit in a better, specifc way

Reply:
IMO we should keep this as we want to archive this repo in near future. When moving this to scancode-toolkit we will not include this file.

Reply:
Done.